optaplanner-docs/src/main/asciidoc/Chapter-Optimization_algorithms/Chapter-Optimization_algorithms.adoc

```diff
@@ -53,41 +52,41 @@ usually perform best for real-world problems given real-world time limitations.
 
 The business wants the optimal solution, but they also have other requirements:
 
-* Scale out: Large production datasets must not crash and have good results too.
+* Scale out: Large production data sets must not crash and have also good results.
 * Optimize the right problem: The constraints must match the actual business needs.
 * Available time: The solution must be found in time, before it becomes useless to execute.
-* Reliability: Every dataset must have at least a decent result (better than a human planner).
+* Reliability: Every data set must have at least a decent result (better than a human planner).
 
-Given these requirements, and despite the promises of some salesmen, it's usually impossible for anyone or anything to find the optimal solution.
+Given these requirements, and despite the promises of some salesmen, it is usually impossible for anyone or anything to find the optimal solution.
 Therefore, Planner focuses on finding the best solution in available time.
 In <<examplesOverview,"realistic, independent competitions">>, it often comes out as the best _reusable_ software.
 
 The nature of NP-complete problems make scaling a prime concern. *The result quality of a
-    small dataset guarantees nothing about the result quality of a large dataset.* Scaling issues cannot be mitigated by hardware purchases later on.
-Start testing with a production sized dataset as soon as possible.
-Don't assess quality on small datasets (unless production encounters only such datasets). Instead, solve a production sized dataset and compare the results of longer executions, different algorithms and - if available - the human planner.
+    small data set guarantees nothing about the result quality of a large data set.* Scaling issues cannot be mitigated by hardware purchases later on.
```

Comment:
This 4 whitespaces at the start and the wierd line ending in the middle of a sentence on the previous line, is a result of the docbook to asciidoc migrator. In the output it doesn't matter, but I think we should clean these up too. So it's always ok to add/remove extra newlines that won't show up in the output. Ideally we 'd want to have no lines in the adoc file that are longer then 120 characters, but that's a soft guideline, not a hard rule.